### PR TITLE
fix(StyledComponents): Updates lib to version 5

### PR DESCRIPTION
**Pull request by renatogalvones**
## Description
https://jirasoftware.catho.com.br/browse/QTM-170

## Setup
- Checkout the repo
```
nvm use;
rm -rf node_modules;
yarn;
yarn storybook;
```

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/Card/__snapshots__/Card.unit.test.jsx.snap [hunk 1]
```diff
@@ -94,6 +94,7 @@ exports[`<Card /> Snapshots Should match the snapshot 1`] = `
     <img
       alt="Thumbnail"
       className="c5"
+      loading="lazy"
```
Comment (tcelestino) [inline components/Card/__snapshots__/Card.unit.test.jsx.snap]: :boom:

File: components/GlobalStyle/sub-components/Typography.jsx [hunk 2]
```diff
@@ -55,21 +54,13 @@ ul, ol {
...
 }`;
 
-const NunitoFontFamily = createGlobalStyle`
-  @import url('https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap');
-`;
-
 const Typography = () => (
   <>
     <Style />
-    {isSSR() ? (
```
Comment (tcelestino) [inline components/GlobalStyle/sub-components/Typography.jsx]: **question:** is there motivation to remove this set to SSR? Now, will CSR and SSR use it?

File: package.json [hunk 3]
```diff
@@ -92,7 +92,7 @@
     "husky": "^1.3.1",
     "jest": "^24.0.0",
     "jest-diff": "^24.0.0",
-    "jest-styled-components": "^6.3.1",
+    "jest-styled-components": "7",
```
Comment (tcelestino) [inline package.json]: would it not be use semantic version here?

File: package.json [hunk 4]
```diff
@@ -104,7 +104,7 @@
     "regenerator-runtime": "^0.13.1",
     "semantic-release": "^15.13.3",
     "strip-ansi": "^5.0.0",
-    "styled-components": "^4.1.3"
+    "styled-components": "5"
```
Comment (tcelestino) [inline package.json]: same here

**Comment by rapahaeru** [comment]
The fonts are different

This PR
image

Current
image

**Comment by rapahaeru** [comment]
Icons not loaded
image

File: components/GlobalStyle/sub-components/Typography.jsx (hunk 2 above)
Comment (rapahaeru) [reply components/GlobalStyle/sub-components/Typography.jsx:65]: Yes, the styled-components > 5 does not support font imports via css, only via html.
https://styled-components.com/releases#v5.0.0

**Pull request merged**